### PR TITLE
Mobile View

### PR DESCRIPTION
**Pull request by Hauwa-Sambo**
Designed a responsive travel landing page using Figma design template.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+           <button id="lock"><i class="material-icons">&#xe897;</i>floattalk.co</button>
+        </div>
+    </div>
+    </nav>
+    <section id="home" class="grid">
+        <div class="home-text"><h2><span style="color: #25a59e;">Travel</span> around the world</h2></div>
+        <div class="home-img">
+            <img src="images/banner_image.png" alt="home-img">
+        </div>
+    </section>
```
Comment (Xfarouq) [inline index.html]: Always make sure your indentation is well

File: style.css [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    color: #1D3066;
+
+}
+#lock{
+    color: #1D3066;
+    font-family: urbanist;
+    font-size: 15px;
+    line-height: 18px;
+    text-align: center;
+width: 356px;
```
Comment (Xfarouq) [inline style.css]: Always be mindful of your indentations

**Review by Xfarouq — CHANGES_REQUESTED** [review]
Hi @Hauwa-Sambo 

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️

- Your design isn't the same as the figma, below are some areas you can focus on 
Screenshot 2024-10-08 134618
Screenshot 2024-10-08 134456


### Optional Suggestions

- Make sure to not leave spaces in your code for best practice
- Always be mindful of your indentations

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+            <h3>Find your best destination</h3>
+            <p>We have more than 1000+ destinations you can choose</p>
+    </header>
+        <section id="destination">
+            <div class="destination grid">
+                <a href="" class="mirror">
+                    <img class="im" src="images/location_Image_1.png" alt="location">
+                    <h4>Mirror Lake</h4>
+                    <p><img src="images/Location.svg" alt="">Jawa Tengah, Indonesia</p>
+                </a>
```
Comment (Devsalmasaeed) [inline index.html]: Be mindful of Html semantic
<section id ="destination">
       <div class="destination grid">
             <div class="mirror">
                     
                      <h4></h4>
                      <p></p>
            </div>
     </div>

File: style.css [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+}
+.last{
+    padding-left: 0px;
+    margin: 0;
+}
+hr.solid{
+    border-top: 1px solid #d9d9d9;
+}
+
+
```
Comment (Devsalmasaeed) [inline style.css]: > Hi @Hauwa-Sambo
> 
> Good job so far! You still need to work on some issues to get your project approved, but you are almost there!
> 
> Remote Collaboration
> ### To Highlight 💯
> * **Clean code** 👍
> * **Good use of Gitflow** 👍
> * **Implemented some project requirements** 👍
> 
> ### Required Changes ♻️
> * Make sure to have professional PR title
> * Your design isn't the same as the figma, below are some areas you can focus on
> Step 2 screenshot
> ### Optional Suggestions
> * Always be mindful of your indentations
> 
> Cheers and Happy coding! 👏👏👏
> 
> _Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification._

**Review by Devsalmasaeed — CHANGES_REQUESTED** [review]

**Review by Xfarouq — APPROVED** [review]
..A

**Review by Xfarouq — APPROVED** [review]
.

**Review by Xfarouq — APPROVED** [review]
Hi @Hauwa-Sambo 

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    Remote Collaboration
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️

List the changes that need to be fixed.
For example,

- kindly consider having the app's name on the splash screen as requested in the project requirement or as given on the design

### Optional Suggestions
Every comment with the `[OPTIONAL]` prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._

**Pull request merged**